_includes/v22.1/sql/select-for-update-example-partial.md

```diff
@@ -0,0 +1,50 @@
+This example assumes you are running a [local unsecured cluster](start-a-local-cluster.html).
+
+First, connect to the running cluster (call this Terminal 1):
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+cockroach sql --insecure
+~~~
+
+Next, create a table and insert some rows:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TABLE kv (k INT PRIMARY KEY, v INT);
+INSERT INTO kv (k, v) VALUES (1, 5), (2, 10), (3, 15);
+~~~
+
+Next, we'll start a [transaction](transactions.html) and lock the row we want to operate on:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+BEGIN;
+SELECT * FROM kv WHERE k = 1 FOR UPDATE;
+~~~
+
+Hit enter twice in the [SQL client](cockroach-sql.html) to send the input so far to be evaluated.  This will result in the following output:
```

Comment:
```suggestion
Press **Enter** twice in the [SQL client](cockroach-sql.html) to send the statements to be evaluated.  This will result in the following output:
```

Reply:
Thanks, updated